linkerd/app/inbound/src/lib.rs

```diff
@@ -42,11 +42,12 @@ pub struct Config {
     pub profile_idle_timeout: Duration,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
```

Comment:
i thought i added a `Debug` impl for the `ErrorRegistry` type? was there a reason we had to remove this?

Reply:
hm, it didn't work for me, but it's also never used afaict